src/java.base/share/classes/jdk/internal/crac/JDKContext.java

```diff
@@ -40,23 +39,14 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.Comparator;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.WeakHashMap;
 
-public class JDKContext extends PriorityContext<JDKResource.Priority, JDKResource> {
+public class JDKContext implements JDKResource {
```

Comment:
Since this is not a context anymore I believe that in the current form the class would deserve a rename (`ClaimedFileDescriptors`?)

Reply:
See other comment https://github.com/openjdk/crac/pull/74#discussion_r1211309286